pkg/common/path_test.go

```diff
@@ -130,6 +145,55 @@ func TestProcessPaths(t *testing.T) {
 	}
 }
 
+func TestExpandFile(t *testing.T) {
+	tf := testutil.Setup(t)
+	defer tf.Clean()
+
+	testCases := map[string]struct {
```

Comment:
Does this change also leverage inventory-objects to see the diff of resources which are deleted from input config after previous apply? An e2e test just covering happy path should be helpful.

Reply:
You're right; an e2e test would be helpful.